packages/moonstone/docs/virtual-list-scroller.md

```diff
@@ -58,19 +53,20 @@ This document describes VirtualList, VirtualGridList, and Scroller.
 
     ```
     //.js
-    renderItem = ({data, index, ...rest}) => {
+    items = []
```

Comment:
; ?

Reply:
We thought this code could be defined outside of class. But it looks this code was defined inside of class. So we're going not to fix it.